lib/topological_inventory/core/service_catalog_client.rb

```diff
@@ -0,0 +1,57 @@
+require "rest_client"
+require "topological_inventory/core/service_plan_client"
+
+module TopologicalInventory
+  module Core
+    class ServiceCatalogClient
+      def initialize(source)
+        @source = source
+      end
+
+      # TODO? Move this to a module in the API repo
+      def order_service_plan(plan_name, service_offering_name, additional_parameters)
+        payload = ServicePlanClient.new.build_payload(
+          plan_name, service_offering_name, catalog_parameters(additional_parameters)
+        )
+        response = make_request(:post, order_service_plan_url, payload)
```

Comment:
@eclarizio 
payload needs to be converted to payload.to_json

Reply:
Done here https://github.com/ManageIQ/topological_inventory-core/pull/39/files#diff-0837d2cfce04474e6a42a58c058ad045R17